tmp/rspec_guard_result

```diff
@@ -0,0 +1,2 @@
+1 examples, 1 failures in 4.0356 seconds
```

Comment:
This file can probably be removed from git and ignored (probably the whole tmp directory). Not a big deal - it'll probably just keep changing and showing up in PRs.